test/bigtest/network/factories/organization.js

```diff
@@ -15,4 +15,10 @@ export default Factory.extend({
   interfaces: [],
   phoneNumbers: [],
   urls: [],
+  metadata: {
+    createdDate: faker.date.past(),
+    createdByUserId: faker.random.uuid,
```

Comment:
can't it introduce an error, when component will try to fetch user that doesn't exist?

Reply:
No, component don't show user name if user doesn't exist